gatsby-node.js

```diff
@@ -155,6 +156,20 @@ exports.onCreatePage = ({ page, actions }) => {
             isPermanent: true,
         })
     }
+    if (is_options) {
+        createRedirect({
+            fromPath: `/trade-types/options/`,
+            toPath: `/trade-types/options/digital-options/up-and-down/`,
+            redirectInBrowser: true,
+            isPermanent: true,
+        })
+        createRedirect({
+            fromPath: `/trade-types/option`,
+            toPath: `/trade-types/options/digital-options/up-and-down/`,
+            redirectInBrowser: true,
+            isPermanent: true,
+        })
+    }
```

Comment:
@mitra-deriv  did we accidentally duplicate this?

Reply:
@prince-deriv `fromPath` is different but not sure whether its correct or not. 👀

Reply:
yes as @fasihali-deriv its different ,  one is with trailing slash and one is without , the same conventions going for other pages we redirecting

Reply:
@mitra-deriv first one has `s` at the end and second one doesn’t contain `s`. 
please check this 😬

Reply:
@fasihali-deriv thanks for noticing bro , fixed 🙏